docs/diagrams/PriorityPSequenceDiagram.puml

```diff
@@ -0,0 +1,56 @@
+@startuml
+!include style.puml
+skinparam ArrowFontStyle plain
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":TaskHubParser" as TaskHubParser LOGIC_COLOR
+participant "p:PriorityProjectCommand" as PriorityProjectCommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+[-> LogicManager : execute(priorityP 1 priority/high)
```

Comment:
Argument in execute should be in double quotes `" "`.

Reply:
Noted!